datasets/superb/superb.py

```diff
@@ -244,6 +284,22 @@ def _generate_examples(self, archive_path, split=None):
                             "text": transcript,
                         }
                         key += 1
+        elif self.config.name == "ks":
+            words = ["yes", "no", "up", "down", "left", "right", "on", "off", "stop", "go"]
+            splits = _split_ks_files(archive_path, split)
+            key = 0
+            for audio_file in sorted(splits[split]):
```

Comment:
As we only have a for loop, maybe it would be simpler to autogenerate the `key` with `enumerate`: see below